### DISCUSSION TITLE
how i get user first name and lastname

### DISCUSSION THREAD

**Discussion by xza-m** [apache/superset#18732] [Q&A / Help]
i want to use jinja get user first name and last name .
i add my def to re:
![8c50a41d7e0424f1bc93aaa93564d04](https://user-images.githubusercontent.com/45376739/154022133-a6fb38d4-deea-4eb6-884f-275e3a842102.png)
my def:
 
![3602e97396e6ba016db5db504b73d2a](https://user-images.githubusercontent.com/45376739/154022254-e1a2904b-3efe-48a0-b80c-3331881200be.png)
add update:
![bfb01989cd7bf2d8f359de6223528ac](https://user-images.githubusercontent.com/45376739/154022300-8f8cf804-7264-4dfa-afe8-3bded385b640.png)
but i test this def error:
![image](https://user-images.githubusercontent.com/45376739/154022366-9d4111ee-e62f-4728-b6f2-b6805c8f36cc.png)

**Comment by IceColdBaby** [comment]
Did you find a way to do it ?
